README.md

```diff
@@ -1,118 +1,80 @@
--	Run this command to start the development environment `docker compose up`, wait 10 seconds and navigate to `localhost`
--   Navigate to Profile menu -> Apps and make sure the "External Storage support" and "External Storage: Swarm" plugins are enabled
--   Navigate to External Storage Administration (Profile menu -> Administration Settings -> External Storage)
--   Add the connection settings for a new Bee node, **make sure** Authentication: none, Configuration is pointing to an active Bee node, and Available for: All users
--   Navigate to Ethswarm Storage Administration (Profile menu -> Administration Settings -> Ethswarm Storage)
--   Configure the Bee node - by default, encryption is active, **make sure** stamp batch is activated by using the toggle
--   To use a Bee node, it is necessary to [Purchase a Batch of new stamps](https://docs.ethswarm.org/docs/access-the-swarm/keep-your-data-alive) from the funds in the node's wallet account
--   Once the new batch is purchased, a unique batchId is generated. Once your batch has been purchased, it will take a few minutes for other Bee nodes in the Swarm to catch up and register your batch. Allow some time for your batch to propagate in the network before proceeding to the next step. This is indicated by the checkbox "Usable".
--   Uploading files to the swarm costs Bzz, so it is necessary to select a Batch as "Active". Only 1 batch is allowed to be Active for a given Bee node. Then click "Save Settings".
+1. Ensure the "External Storage" plugin is enabled.
+2. Install the "External Storage: Hejbit Swarm" app from the Nextcloud App Store.
+3. Navigate to External Storage Administration: Profile menu -> Settings -> External Storage.
+4. Create a new external storage with a folder name of your choice and select 'hejbit-swarm' as the storage type. Then, enter the license key and URL configuration.
 
 ## Usage
 
-File handling operations involving the Bee node are integrated in the External Storage ("Files") section of Nextcloud.
-
-![View file and action menu](assets/images/swarm_Files_Intro.png "View file and action menu")
-
-<em>View, download, and file view actions in Nextcloud</em>
+Once installed, Hejbit Swarm integrates directly with Nextcloud’s "Files" section, allowing seamless file operations:
 
-Using the plugin, the main file operations that are available in Nextcloud are:
+- **Upload:** Transfer files to decentralized storage.
+- **Download:** Retrieve and view files from decentralized storage.
+- **Copy/Move:** Shift files between any external storage and decentralized storage
 
--   Upload file(s) to a Bee node
--   Download and view files
--   Copy/Move files from any External Storage to a Bee node
--   Copy files from a Bee node to any External Storage
--   View and copy the unique swarm reference to the clipboard for any file stored on a Bee node
+![View, download, and file view actions in Nextcloud](https://github.com/user-attachments/assets/3bee08e4-7a9e-4b44-a904-821359cc3e7b)
```

Comment:
This picture is not up-to-date, we don't have the port number no more.

Reply:
@frozenbanana I agree, we can update the pictures on the next release.